### ISSUE TITLE
Code of conduct and/or contributing guide

### ISSUE THREAD

**Issue by akirabaruah** [cardinalitypuzzles/smallboard#236]
As we have more contributors working on this project, it'll be helpful to clearly define the basic steps to get involved via Github Issues. We should probably encourage discussion in Issues and ask folks to mention that they're working on something so we can avoid duplicated work.

This doc doesn't have to include technical details about getting a local dev setup going (that can stay in the existing dev guide), but perhaps an overview of the typical way that folks (especially newer contributors) can get involved.

What do folks think?

**Comment by erwa** [comment]
I'm supportive of adding one though I don't think we _really_ need one at this point and can mentor and ramp newcomers up on an adhoc basis. That said, we could probably just add a contributing.md guide in the repo itself. (We could also use GitHub's Wiki functionality, but I think it's better if guides/documentation live in the repo where they can be more easily versioned and kept up-to-date and also more easily "code reviewed")

**Comment by akirabaruah** [comment]
Yeah I think just adding a contributing.md to the repo should be sufficient. I was envisioning a doc that we could point new contributors to when they're getting started so we don't have to repeat common steps, advice, etc.
